scripts/README.md

```diff
@@ -0,0 +1,9 @@
+# Code Generation
```

Comment:
This is great, we may want to mention `methodsWithCustomTypes` but we can add that in a follow-up PR